builtin/providers/gitlab/resource_gitlab_project.go

```diff
@@ -91,6 +95,7 @@ func resourceGitlabProjectCreate(d *schema.ResourceData, meta interface{}) error
 	client := meta.(*gitlab.Client)
 	options := &gitlab.CreateProjectOptions{
 		Name:                 gitlab.String(d.Get("name").(string)),
+		NamespaceID:          gitlab.Int(d.Get("namespace_id").(int)),
```

Comment:
One change needed here, this is optional, so it needs to move from here and go with the d.GetOk("") patterned stuff:

~~~
if v, ok := d.GetOk("namespace_id"); ok {
   options.NamespaceID = gitlab.Int(v.(int))
}
~~~